### ISSUE TITLE
CVE in go modules in used by traefik

### ISSUE THREAD

**Issue by jamesocIRL** [traefik/traefik#8604]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.containo.us) and didn't find any.

### What did you do?

I ran a trivy scan on the latest release. I expected not to see any cves listed.

### What did you see instead?
Listed CVE:
cve-2021-41103
[cve-2021-41092](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-41092)

```
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -it aquasec/trivy:latest image --severity HIGH,CRITICAL traefik:v2.5.4
2021-12-03T15:47:29.233Z        INFO    Need to update DB
2021-12-03T15:47:29.233Z        INFO    Downloading DB...
25.08 MiB / 25.08 MiB [----------------------------------------------] 100.00% 426.48 KiB p/s 1m1s
2021-12-03T15:48:32.399Z        INFO    Detected OS: alpine
2021-12-03T15:48:32.399Z        INFO    Detecting Alpine vulnerabilities...
2021-12-03T15:48:32.399Z        INFO    Number of language-specific files: 1
2021-12-03T15:48:32.399Z        INFO    Detecting gobinary vulnerabilities...

traefik:v2.5.4 (alpine 3.14.3)
==============================
Total: 0 (HIGH: 0, CRITICAL: 0)


usr/local/bin/traefik (gobinary)
================================
Total: 2 (HIGH: 2, CRITICAL: 0)

+----------------------------------+------------------+----------+------------------------------------+-----------------+---------------------------------------+
|             LIBRARY              | VULNERABILITY ID | SEVERITY |         INSTALLED VERSION          |  FIXED VERSION  |                 TITLE                 |
+----------------------------------+------------------+----------+------------------------------------+-----------------+---------------------------------------+
| github.com/containerd/containerd | CVE-2021-41103   | HIGH     | v1.3.2                             | v1.4.11, v1.5.7 | containerd: insufficiently            |
|                                  |                  |          |                                    |                 | restricted permissions on container   |
|                                  |                  |          |                                    |                 | root and plugin directories           |
|                                  |                  |          |                                    |                 | -->avd.aquasec.com/nvd/cve-2021-41103 |
+----------------------------------+------------------+          +------------------------------------+-----------------+---------------------------------------+
| github.com/docker/cli            | CVE-2021-41092   |          | v0.0.0-20200221155518-740919cc7fc0 | v20.10.9        | docker: cli leaks private registry    |
|                                  |                  |          |                                    |                 | credentials to registry-1.docker.io   |
|                                  |                  |          |                                    |                 | -->avd.aquasec.com/nvd/cve-2021-41092 |
+----------------------------------+------------------+----------+------------------------------------+-----------------+---------------------------------------+
```

### What version of Traefik are you using?

v2.5.4

### What is your environment & configuration?

NA

**Comment by ldez** [comment]
Hello,

it's already fixed by #8593

**Comment by jamesocIRL** [comment]
Oh I am sorry, searched by CVE and somehow missed it! Great job and thank you!
